File: Java/Logic - 2/roundSum.java
public int roundSum(int a, int b, int c) {

    return round10(a) + round10(b) + round10(c);

}

public int round10(int num) {

    if(num%10 >= 5) {

        return (num/10+1)*10;

    }

    else {

        return (num/10) * 10;

    }

}

File: Python/Logic - 2/round_sum.py
def round_sum(a, b, c):
    
    return round10(a) + round10(b) + round10(c)

def round10(n):
    
    if n%10 >= 5:
        
        return (n//10)*10+10
    
    else:
        
        return (n//10)*10